control/content/app.js

```diff
@@ -15,7 +15,7 @@
                     controllerAs: 'ContentChat',
                     controller: 'ContentChatCtrl'
                 })
-                .when('/chat/:userToken', {
+                .when('/chat/:encodedReview', {
```

Comment:
- Use the framework's state to store the reviews collection.
- Then access the target review by sending its userId as a query parameter.
- Manage the framework's state (update/delete/create)

Reply:
`/chat/:userId`